exercises/nodejs/src/board.js

```diff
@@ -0,0 +1,31 @@
+import {
```

Comment:
I think this is a great change
- Encapsulating the Board knowledge into a seperate class
  - Makes reading the board functionality a lot easier, e.g. how the game processes categories